wallet/src/ui/app/hooks/useMediaUrl.ts

```diff
@@ -10,7 +10,7 @@ export default function useMediaUrl(objData: SuiData) {
     const { fields } = (isSuiMoveObject(objData) && objData) || {};
     return useMemo(() => {
         if (fields) {
-            const mediaUrl = fields.url;
+            const mediaUrl = fields.url || fields.metadata?.fields.uri;
```

Comment:
we should also consider using `url` rather than `uri`

Reply:
We are changing the name of the field to `url` in this [PR](https://github.com/Origin-Byte/nft-protocol/pull/8)

Reply:
The PR has been merged, from version `0.3.0` onwards it is now `url `

Reply:
Field name has been changed as well.